javascript/javascript2/week3/warmup.js

```diff
@@ -0,0 +1,93 @@
+
+window.addEventListener("load", () => {
+  setTimeout(() => {
+    console.log("load efter 2,5 seconds");
+  }, 2500);
+});
+
+// function with delay parameter
+let delayfunction = (stringToLog,delay) =>{
```

Comment:
```suggestion
const delayfunction = (stringToLog,delay) =>{
```

Always use `const` until you can't. Only use `let` when you need to re-assign a variable.